README.md

```diff
@@ -68,7 +68,7 @@ The project welcomes submissions, but please let everyone know what you are work
 
 Before undertaking a nontrivial change to this specification, send mail to the [mailing list](#mailing-list) to discuss what you plan to do.
 This gives everyone a chance to validate the design, helps prevent duplication of effort, and ensures that the idea fits.
-It also guarantees that the design is sound before code is written; a GitHub pull-request is not the place for high-level discussions.
+It also guarantees that the design is sound before code is written, a GitHub pull-request is not the place for high-level discussions.
```

Comment:
I think the semicolon was right here.

The earlier LICENSE link looks good to me.